ios/ExpoNotificationServiceExtension/Info.plist

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>$(DEVELOPMENT_LANGUAGE)</string>
+	<key>CFBundleDisplayName</key>
+	<string>ExpoNotificationServiceExtension</string>
```

Comment:
```suggestion
	<string>ExpoGoNotificationServiceExtension</string>
```

Reply:
On second thought, CFBundleDisplayName is often used for a user-facing name. Does this string show up in iOS anywhere?

Reply:
from apple:

> An app extension needs a short, recognizable name that includes the name of the containing app, using the pattern `<Containing app name>—<App extension name>`. This makes it easier for users to manage extensions throughout the system.

So `ExpoGoNotificationServiceExtension` follows their recommended format. + I can't find anywhere that this value is user-facing (maybe that only applies to _some_ app extensions?)

Reply:
It sounds like their recommendation is saying to name it `Expo Go—Notifications`